_docs/_latest/configure/scanner/rules.md

```diff
@@ -231,7 +233,7 @@ To enable the blacklist notifier, add the followings to the notifier section in
         - resource: blacklist_violations
           should_notify: true
           pipelines:
-            # Upload violations to GCS.
+            # Upload violations to Cloud Storage.
```

Comment:
@joecheuk and @blueandgold can we ensure the sample configuration and default rules use this same naming format in its comments?

Reply:
This section is not needed any more, can we remove the two `Enabling` sections for Blacklist scanner and Kubernetes scanner? We have the updated information in other files under configure/

Reply:
Done!